Source/Scene/Model.js

```diff
@@ -1719,6 +1741,19 @@ define([
         return shader;
     }
 
+    function modifyShaderForBlendColor(shader) {
+        shader = ShaderSource.replaceMain(shader, 'czm_blend_main');
+        shader +=
+            'uniform vec4 czm_blendColor; \n' +
```

Comment:
Instead of `czm_`, should we use something like `gltf_` (like we use `tile_` in 3D Tiles)?  I probably used `czm_` elsewhere for 3D models, but perhaps we should reserve it for built-ins.